util/csrankings.py

```diff
@@ -223,8 +223,22 @@ def test_pagecount():
         Conference("EMNLP-IJCNLP"),
         Conference("EMNLP/IJCNLP (1)"),
     ],
-    Area("acl"): [Conference("ACL"), Conference("ACL (1)"), Conference("ACL (2)"), Conference("ACL/IJCNLP"), Conference("COLING-ACL")],
-    Area("naacl"): [Conference("NAACL"), Conference("HLT-NAACL"), Conference("NAACL-HLT"), Conference("NAACL-HLT (1)")],
+    Area("acl"): [
+        Conference("ACL"), 
+        Conference("ACL (1)"), 
+        Conference("ACL (2)"), 
+        Conference("ACL/IJCNLP"), 
+        Conference("ACL/IJCNLP (1)"),
+        Conference("ACL/IJCNLP (2)"),
+        Conference("COLING-ACL")
+    ],
+    Area("naacl"): [
+        Conference("NAACL"), 
+        Conference("HLT-NAACL"), 
+        Conference("NAACL-HLT"), 
+        Conference("NAACL-HLT (1)"),
+        Conference("NAACL-HLT (Industry Papers)"), 
```

Comment:
This needs to be removed; we don't include separate industry track papers.

Reply:
bump @lileicc

Reply:
Fixed. 
But you may want to state clearly what is included and what is not. Right now, the only rule is in those listed conference and with more than 6 pages. If certain proceedings are not included for a particular conference, you may want to state it and explain it clearly the rationale. Otherwise it is confusing -- industry papers are included for some conference but not for others. Thanks!

Reply:
Thanks.
As far as I am aware, industry-track papers — where this means papers treated differently from ordinary research papers — are not counted by CSrankings. Certainly this is the intent. If you are aware of cases where these are being inadvertently included, I'd like to hear about them.